mne/io/edf/edf.py

```diff
@@ -77,6 +77,9 @@ class RawEDF(BaseRaw):
 
     It is also possible to retrieve system codes, but no particular effort has
     been made to decode these in MNE.
+    For GDF files, the stimulus channel is constructed from the events in the
+    header. You should use keyword ``stim_channel=-1`` to add it at the end of
+    the channel list.
```

Comment:
I thought about having 'auto', but that could potentially break things if there happens to be a channel named 'auto'.